lib/ping.ts

```diff
@@ -24,7 +24,7 @@ export function startPingJob (
       log.info("", {
         itasCorrelationId,
         name: 'Sending request',
-        upstreamURL,
+        url: upstreamURL + '/pong',
```

Comment:
Consider adding this as a const URL (https://nodejs.org/api/url.html#url_url) and use the same url for both logging and the actual request

Reply:
Yeah I was 50/50 if I wanted to do it, but I guess it is a good idea.

However, for this simple case I'd like to just use a string, as I don't want to read documentation for a URL library when all I need is to reuse a string.